packer/scylla_install_image

```diff
@@ -78,11 +80,14 @@ if __name__ == '__main__':
         print('no scylla package found.')
         sys.exit(1)
 
+    run(f'wget -O /etc/apt/sources.list.d/scylla-manager.list ${args.scylla_manager_repo}', shell=True, check=True)
+
     run('apt-get update --allow-insecure-repositories -y', shell=True, check=True)
     run('apt-get full-upgrade -y', shell=True, check=True)
     run('apt-get purge -y apport python3-apport fuse', shell=True, check=True)
     run('apt-get install -y systemd-coredump', shell=True, check=True)
     run(f'apt-get install -y --auto-remove --allow-unauthenticated {args.product}-machine-image {args.product}-server-dbg', shell=True, check=True)
+    run(f'apt-get install -y scylla-manager-agent', shell=True, check=True)
```

Comment:
hopefully it won't need SCT to change, but I think the artifact test would cover it. (i.e. it would try to install a different version, based on SCT logic)

Reply:
@fruch Already spoke with @ShlomiBalalis  and @fgelcer about this change, @ShlomiBalalis Will open a task for him to do the needed changes